<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Freparation</class>
 <widget class="QDialog" name="Freparation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reparation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rssources.qrc">
    <normaloff>:/images/logo_erp.png</normaloff>:/images/logo_erp.png</iconset>
  </property>
  <widget class="QPushButton" name="confirm_but">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmer</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_repair_text">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupe_actes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>291</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Actes</string>
   </property>
   <widget class="QLineEdit" name="panne_text">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Panne</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cout_text">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cout</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Produit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_prod">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_etat">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>En marche</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bloqu√©</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>En attente</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="rssources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Freparation</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
